crates/rslint_lexer/src/lib.rs

```diff
@@ -757,6 +757,32 @@ impl<'src> Lexer<'src> {
         }
     }
 
+    #[inline]
+    fn read_shebang(&mut self) -> LexerReturn {
+        assert_eq!(self.cur, 0);
+        self.next();
+        if let Some(b"#!") = self.bytes.get(0..2) {
+            while self.next().is_some() {
+                let chr = self.get_unicode_char();
+                self.cur += chr.len_utf8() - 1;
+
+                if is_linebreak(chr) {
+                    break;
+                }
```

Comment:
This should follow the same logic as single line comments to be consistent and avoid bugs which have been caught
https://github.com/RDambrosio016/RSLint/blob/master/crates/rslint_lexer/src/lib.rs#L790